examples/server-nonblocking.lua

```diff
@@ -0,0 +1,40 @@
+-- REMOVE NEXT LINE BEFORE MERGING
+package.path = ';./src/?/init.lua;./src/?.lua;/usr/local/share/lua/5.1/?.lua'
```

Comment:
Looks like you intended on removing this before creating the PR?

Reply:
I will remove it in an upcoming commit, like the comment and the PR says this is just a quick draft I put together for @davidgranstrom and you to review, thanks
But I guess eveyone installs with `luarocks make`and this was not even needed in the first place!